reading_order_widget.py

```
@@ -50,159 +63,348 @@ def __init__(self):
+        drag.setPixmap(scaled)
+        drag.setHotSpot(QPoint(scaled.width() // 2, scaled.height() // 2))
+        drag.exec(Qt.DropAction.CopyAction)
+
     def dragEnterEvent(self, event) -> None:
+        """
+        Allows a drag action from this widget, or from the DraggableComicGridView.
+        """
         if event.mimeData().hasFormat(self.MIME_TYPE):
             event.acceptProposedAction()
+        elif event.mimeData().hasFormat(DraggableComicGridView.MIME_TYPE):
+            event.acceptProposedAction()
```

Ruff (0.15.12)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)